app/assets/javascripts/components/stories/Stories.js

```diff
@@ -1,20 +1,23 @@
-import React, { Fragment } from "react";
-import StoryItem from "../story/StoryItem";
+import React from 'react';
+import StoryItem from '../story/StoryItem';
 import PropTypes from 'prop-types';
 import { storyPropTypesShape } from './../../models/beta/story';
 
-const Stories = ({ stories, from }) => {
+const Stories = ({ column, stories, from }) => {
```

Comment:
Maybe refactor this component with `ternary operator` and `functional component`. What you think @talyssonoc ?

```
const Stories = ({ column, stories, from }) =>
  Story.hasNoStories(stories) 
    ? null
    : stories.map((story, index) => (
        <StoryItem
          key={story.id}
          index={index}
          stories={stories}
          story={story}
          column={column}
          from={from}
        />
      ));
```